packer.mk

```diff
@@ -49,3 +49,7 @@ convert-sig-to-classic-storage-account-blob: az-login
 
 windows-vhd-publishing-info: az-login
 	@./vhdbuilder/packer/generate-windows-vhd-publishing-info.sh
+
+test-building-vhd: az-login
```

Comment:
wanted to put this target to `test.mk`, but move back here for:
1. `test.mk` does not have `az login` to authenticate, we may share the common target in common.mk in the future
2. `test-building-vhd` is also part of packer to validate the result of packer work.
Happy to discuss.